install.sh

```diff
@@ -177,3 +177,4 @@ fi
 # Installation Complete
 echo
 echo [~] Installation Complete....
+echo [~] Reboot the terminal to affect changes.
```

Comment:
Probably you should use "restart" the terminal.

Reply:
Alternatively, why don't you integrate this answer into the shell script: https://unix.stackexchange.com/questions/217905/restart-bash-from-terminal-without-restarting-the-terminal-application-mac